### PR TITLE
feat: Continue Get end point

### PR DESCRIPTION
**Pull request by Ansaar-Syed**
## Pre-Requisites
- [x] Yes, I updated [Authors.md](../blob/main/Authors.md) **OR** this is not my first contribution
- [x] Yes, I included and/or modified tests to cover relevent code **OR** my change is non-technical
- [x] Yes, I wrote this code entirely myself **OR** I properly attributed these changes in [Third Party Notices](../blob/main/THIRD-PARTY-NOTICES.txt)

### CODE REVIEW

File: packages/api/src/api/expoJudgingSession/id/continue/index.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express';
+import { get } from './get';
+
+export const Continue = Router();
```
Comment (charliealbright) [inline packages/api/src/api/expoJudgingSession/id/continue/index.ts:4]: Let's lowercase the `continue` export so it matches other exported routes. If you do a rename in the editor it should fix it across all of your changes here 👍🏻

File: packages/api/src/api/expoJudgingSession/id/index.ts [hunk 2]
```diff
@@ -2,9 +2,11 @@ import { Router } from 'express';
...
 import { projects } from './projects';
 import { skip } from './skip';
+import { Continue } from './continue';
 
 export const id = Router({ mergeParams: true });
 
 id.get('', get);
 id.use('/projects', projects);
 id.use('/skip', skip);
+id.use('/Continue', Continue);
```
Comment (charliealbright) [inline packages/api/src/api/expoJudgingSession/id/index.ts:12]: Make sure to lowercase this path to `'/continue'`

**Review by charliealbright — COMMENTED** [review]
Couple small things to address, otherwise lgtm

File: packages/api/src/api/expoJudgingSession/id/continue/index.ts (hunk 1 above)
Comment (Ansaar-Syed) [reply packages/api/src/api/expoJudgingSession/id/continue/index.ts:4]: i tried doing that it gives an error " 'continue' is not allowed as a variable declaration name."

File: packages/api/src/api/expoJudgingSession/id/index.ts [hunk 3]
```diff
@@ -2,9 +2,11 @@ import { Router } from 'express';
 import { get } from './get';
 import { projects } from './projects';
 import { skip } from './skip';
+import { resume } from './resume';
```
Comment (charliealbright) [inline packages/api/src/api/expoJudgingSession/id/index.ts:5]:
```suggestion
import { continueSession } from './continueSession';
```

File: packages/api/src/api/expoJudgingSession/id/index.ts [hunk 4]
```diff
@@ -2,9 +2,11 @@ import { Router } from 'express';
...
 
 export const id = Router({ mergeParams: true });
 
 id.get('', get);
 id.use('/projects', projects);
 id.use('/skip', skip);
+id.use('/continue', resume);
```
Comment (charliealbright) [inline packages/api/src/api/expoJudgingSession/id/index.ts:12]:
```suggestion
id.use('/continue', continueSession);
```

File: packages/api/src/api/expoJudgingSession/id/resume/index.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { Router } from 'express';
+import { get } from './get';
+
+export const resume = Router();
+
+resume.get('', get);
```
Comment (charliealbright) [inline packages/api/src/api/expoJudgingSession/id/resume/index.ts:6]:
```suggestion
export const continueSession = Router();

continueSession.get('', get);
```

**Review by charliealbright — COMMENTED** [review]
Since we're going to leave the API path as `/continue`, let's change our strategy here. Use `continueSession` as your variable names to get around the compiler keyword restriction. Don't forget to change file/folder names back to `continue`

**Review by charliealbright — APPROVED** [review]

**Pull request merged**